script/preview.js

```diff
@@ -84,6 +84,23 @@ if (process.env.SENTRY_DSN) {
   app.use(Raven.requestHandler());
 }
 
+/**
+ * Make the query params case-insensitive.
+ */
+app.use((req, res, next) => {
+  // eslint-disable-next-line fp/no-proxy
+  req.query = new Proxy(req.query, {
```

Comment:
Can we assign that proxy to something like `caseInsensitiveQueryParams` before assigning `req.query`? Idk, i personally prefer that over comments

Reply:
If we do that, would we / should we ever use `req.query`?

Reply:
Oh no, I just mean assigning the proxy and then `req.query = caseInsensitiveQueryParams`.  But it doesn’t really bother me either way.